### DISCUSSION TITLE
Custom build with older libreOffice ver

### DISCUSSION THREAD

**Discussion by benben-user** [gotenberg/gotenberg#1025] [Q&A]
Hi , will appreciate some help with downgrade libreOffice in a custom build. 
Use Case:
New 24.8 ver convert nested tables different then 7.x , Gotenberg instance get timeout and not able to restart.

**Comment by benben-user** [comment]
@gulien Can you provide some insights ?

**Reply by gulien** [reply]
Hello @benben-user, best option is to check the Dockerfile to understand how it is currently done and how to modify it. I don't have enough time to provide more in depth highlight.
